From: Scott Kitterman <scott@kitterman.com>
Date: Fri, 24 Dec 2021 13:42:26 -0500
Subject: Patch away setup requires

Debian python stuff takes care of this and does it differently.

Origin: vendor
Forwarded: not-needed
Last-Update: 2021-12-24
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 0e28c67..59f364f 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -46,7 +46,7 @@ packages =
     dns.rdtypes.CH
 python_requires = >=3.7
 test_suite = tests
-setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
+#setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
 
 [options.extras_require]
 DOH = httpx>=0.21.1; h2>=4.1.0; requests; requests-toolbelt
